bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -819,7 +819,10 @@ public static Long win32_getHandle (Image image, int zoom) {
 			image.init(newData, zoom);
 		}
 	} else if (image.imageDataProvider != null) {
-		ElementAtZoom<ImageData> imageCandidate = DPIUtil.validateAndGetImageDataAtZoom (image.imageDataProvider, zoom);
+		ElementAtZoom<ImageData> imageCandidate;
+		try (StaticZoomUpdater ignored = new StaticZoomUpdater(zoom)) {
```

Comment:
The name `ignored` confused me. I guess it is meant to indicate that the variable itself is "ignored", but that was not clear to me at first. Maybe rather call it `unused` or simply `__` (although I know there is some controversy on whether that's good practice).

Reply:
Sure, I will change that one

Reply:
I updated it